app/config/messagingProviders.php

```diff
@@ -0,0 +1,300 @@
+<?php
+
+return [
```

Comment:
Let's remove any providers that are not currently supported in the messaging library, as well as format this file with 4 spaces